kit/brain_board/python_libraries.md

```diff
@@ -3,74 +3,47 @@ layout: page
-* [cycler 0.11.0](https://pypi.org/project/cycler)
-* [dateutil 2.8.2](https://pypi.org/project/python-dateutil)
-* [debugpy 1.6.3](https://pypi.org/project/debugpy)
-* [dbus-next 0.2.3](https://pypi.org/project/dbus-next)
-* [docopt 0.6.2](https://pypi.org/project/docopt)
-* [gmqtt 0.6.11](https://pypi.org/project/gmqtt)
-* [gpg 1.17.1](https://pypi.org/project/gpg)
-* [iniparse 0.5](https://pypi.org/project/iniparse)
-* [j5 1.1.2](https://pypi.org/project/j5)
-* [kchd 0.4.1](https://pypi.org/project/kchd)
-* [matplotlib 3.5.1](https://pypi.org/project/matplotlib)
-* [numpy 1.22.3](https://pypi.org/project/numpy)
-* opencv 4.5.5
-* [paho-mqtt 1.6.1](https://pypi.org/project/paho-mqtt)
-* [pandas 1.4.2](https://pypi.org/project/pandas)
-* [Pillow 9.0.1](https://pypi.org/project/Pillow)
-* [prompt-toolkit 3.0.24](https://pypi.org/project/prompt-toolkit)
-* [pyapriltags 3.3.0.post2](https://pypi.org/project/pyapriltags)
-* [pydantic 1.9.1](https://pypi.org/project/pydantic)
-* [PyGObject 3.42.0](https://pypi.org/project/PyGObject)
-* [pyparsing 3.0.7](https://pypi.org/project/pyparsing)
-* [pyquaternion 0.9.9](https://pypi.org/project/pyquaternion)
-* [pyserial 3.5](https://pypi.org/project/pyserial)
-* [pytz 2022.1](https://pypi.org/project/pytz)
-* [pyudev 0.23.2](https://pypi.org/project/pyudev)
-* [pyusb 1.2.1](https://pypi.org/project/pyusb)
-* [pyyaml 6.0](https://pypi.org/project/PyYAML)
-* [RPi.GPIO 0.7.0](https://pypi.org/project/rpi.gpio)
-* [rtui 0.1.1](https://pypi.org/project/rtui)
-* [setuptools 59.5.0](https://pypi.org/project/setuptools)
-* [six 1.16.0](https://pypi.org/project/six)
-* [sr.robot3 2023.2.0](https://pypi.org/project/sr.robot3)
-* [tomli 2.0.1](https://pypi.org/project/tomli)
-* [tomli-w 1.0.0](https://pypi.org/project/tomli-w)
-* [typing-extensions 3.10.0.0](https://pypi.org/project/typing-extensions)
-* [udiskie 2.3.3](https://pypi.org/project/udiskie)
+* [debugpy 1.7.0](https://pypi.org/project/debugpy)
+* [flask 2.3.3](https://pypi.org/project/flask)
+* [matplotlib 3.7.2](https://pypi.org/project/matplotlib)
+* [networkx 3.1](https://pypi.org/project/networkx)
+* [numpy 1.24.4](https://pypi.org/project/numpy)
+* [opencv-python-headless 4.8.0.76](https://pypi.org/project/opencv-python-headless)
+* [pandas 2.0.3](https://pypi.org/project/pandas)
+* [pillow 10.0.0](https://pypi.org/project/pillow)
+* [scikit-learn 1.3.0](https://pypi.org/project/scikit-learn)
+* [scipy 1.10.1](https://pypi.org/project/scipy)
+* [shapely 2.0.1](https://pypi.org/project/shapely)
+* [sr-robot3 2024.0.0](https://pypi.org/project/sr-robot3)
 <!-- cspell:enable -->
 
-If you wish to use a library that isn't listed above, get in contact with us on Discord and have a chat with us about it.
 
+## Simulator differences
 
-## Simulator
+Generally we will try to keep the libraries available in the simulator the same as what is available on the physical robot.
+The list below outlines the differences.
```

Comment:
Do we want to note here that the `sr-robot3` available in the simulator isn't the one from PyPI?

Reply:
Hows this c850300